stl/inc/format

```diff
@@ -46,6 +46,11 @@ concept _Parse_spec_callbacks = requires(_Ty _At, basic_string_view<_CharT> _Sv,
     { _At._On_width(_STD declval<int>()) } -> same_as<void>;
     { _At._On_dynamic_width(_STD declval<int>()) } -> same_as<void>;
     { _At._On_dynamic_width(_STD declval<_Auto_id_tag>()) } -> same_as<void>;
+    { _At._On_precision(_STD declval<int>()) } -> same_as<void>;
+    { _At._On_dynamic_precision(_STD declval<int>()) } -> same_as<void>;
+    { _At._On_dynamic_precision(_STD declval<_Auto_id_tag>()) } -> same_as<void>;

```

Comment:
Pre-existing, no change requested: `declval` is important for generic code where we don't know how to construct the type. For types we know/control, it might be simpler to say `int{}` and `_Auto_id_tag{}`. I don't know what conventions @CaseyCarter has been using in concepts. Note that the rules for null pointer constants have been refined so `int{}` is ineligible, WG21-N4861 7.3.11 [conv.ptr]/1 "A *null pointer constant* is an integer literal with value zero or a prvalue of type `std::nullptr_t`."

Reply:
DOH! it didn't even occur to me that int{} was something I could write here.

Reply:
I added a project card to your To Do column since I didn't see one there.